docs/functions/lock.md

```diff
@@ -0,0 +1,33 @@
+## @lock()
+
+Executes the decorated function as soon as you call it for the first
```

Comment:
I think we should update the description of the new decorator here

Reply:
Updated!